CrashReporting/security-privacy.md

```diff
@@ -0,0 +1,39 @@
+### Questions from https://www.w3.org/TR/security-privacy-questionnaire/
+
+# 2. Questions to Consider
+## 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+This feature exposes JavaScript call stacks at the time of a crash to website developers. This information is necessary for diagnosing and resolving issues that may cause a page to become unresponsive due to JavaScript execution.
+## 2.2. Do features in your specification expose the minimum amount of information necessary to enable their intended uses?
+Yes, the feature is designed to include JavaScript call stacks and only when developers opt-in to receive them.
+## 2.3. How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
+This feature does not expose any personal or personally identifiable information. It only includes JavaScript call stacks at the time of the crash.
+## 2.4. How do the features in your specification deal with sensitive information?
+The feature does not deal with sensitive information.
+## 2.5. Do the features in your specification introduce new state for an origin that persists across browsing sessions?
+No, the feature does not introduce new persistent state for an origin. It only reports the JavaScript call stacks at the time of a crash, which does not persist across sessions.
+## 2.6. Do the features in your specification expose information about the underlying platform to origins?
```

Comment:
I think this should mention the extension issue, where it can be observed that a user is running a given extension if a crash is reported with frames from the extension on the stack.

Reply:
and you're suggesting I add this to question 2.6 right?

Reply:
Yes